signing/src/test/java/tech/pegasys/web3signer/signing/secp256k1/azure/AzureKeyVaultSignerTest.java

```diff
@@ -28,54 +29,51 @@
 import org.junit.jupiter.api.Assumptions;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-import org.web3j.crypto.Hash;
 import org.web3j.crypto.Sign;
 import org.web3j.crypto.Sign.SignatureData;
 import org.web3j.utils.Numeric;
 
 public class AzureKeyVaultSignerTest {
-  private static final String clientId = System.getenv("AZURE_CLIENT_ID");
-  private static final String clientSecret = System.getenv("AZURE_CLIENT_SECRET");
-  private static final String keyVaultName = System.getenv("AZURE_KEY_VAULT_NAME");
-  private static final String tenantId = System.getenv("AZURE_TENANT_ID");
-  // uses curve name P-256K
-  private static final String KEY_NAME = "TestKey2";
+  private static final String AZURE_CLIENT_ID = System.getenv("AZURE_CLIENT_ID");
+  private static final String AZURE_CLIENT_SECRET = System.getenv("AZURE_CLIENT_SECRET");
+  private static final String AZURE_KEY_VAULT_NAME = System.getenv("AZURE_KEY_VAULT_NAME");
+  private static final String AZURE_TENANT_ID = System.getenv("AZURE_TENANT_ID");
+  private static final String AZURE_INVALID_KEY_VAULT_NAME =
+      System.getenv("AZURE_INVALID_KEY_VAULT_NAME");
+
+  private static final String KEY_NAME = "TestKey2"; // uses curve name P-256K
   private static final String UNSUPPORTED_CURVE_KEY_NAME = "TestKeyP521";
 
   @BeforeAll
   static void preChecks() {
     Assumptions.assumeTrue(
-        clientId != null && clientSecret != null && keyVaultName != null && tenantId != null,
+        AZURE_CLIENT_ID != null
+            && AZURE_CLIENT_SECRET != null
+            && AZURE_KEY_VAULT_NAME != null
+            && AZURE_INVALID_KEY_VAULT_NAME != null
+            && AZURE_TENANT_ID != null,
         "Ensure Azure env variables are set");
   }
 
   @Test
-  public void azureSignerCanSignTwice() {
+  void azureSignerCanSign() throws SignatureException {
     final AzureConfig config =
-        new AzureConfig(keyVaultName, KEY_NAME, "", clientId, clientSecret, tenantId);
-
-    final AzureKeyVaultSignerFactory factory = new AzureKeyVaultSignerFactory();
-    final Signer signer = factory.createSigner(config);
-
-    final byte[] dataToHash = "Hello World".getBytes(UTF_8);
-    signer.sign(dataToHash);
-    signer.sign(dataToHash);
-  }
-
-  @Test
-  void azureWithoutHashingDoesntHashData() throws SignatureException {
-    final AzureConfig config =
-        new AzureConfig(keyVaultName, KEY_NAME, "", clientId, clientSecret, tenantId);
+        new AzureConfig(
+            AZURE_KEY_VAULT_NAME,
+            KEY_NAME,
+            "",
```

Comment:
We could either add a comment or a static with a name that tells what this is for the sake of readability

Reply:
Changed to use the new constructor that doesn't require the key version